eng/scripts/Create-ApiReview.ps1

```diff
@@ -28,10 +28,10 @@ Set-StrictMode -Version 3
 . (Join-Path $PSScriptRoot ".." common scripts common.ps1)
```

Comment:
Why does python have a script not under eng/common?

Reply:
Python uses a pre generated API review token file and a different API request when we started sandboxing change. I will check if I can merge this to single script which supports both sandboxing and upload. I will file an issue to track it.